flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -98,6 +98,13 @@ inline mlir::Type getVoidPtrType(mlir::MLIRContext *context) {
   return mlir::LLVM::LLVMPointerType::get(mlir::IntegerType::get(context, 8));
 }
 
+static mlir::LLVM::ConstantOp
+genConstantIndex(mlir::Location loc, mlir::Type ity,
+                 mlir::ConversionPatternRewriter &rewriter, int offset) {
```

Comment:
Should `offset` be `std::int64_t` type?

Reply:
It's moved code with no change, but I agree it makes sense. Done.